Sources/StreamChat/Utils/Database/NSManagedObject+Extensions.swift

```diff
@@ -10,13 +10,109 @@ extension NSManagedObject {
     }
 }
 
+extension NSEntityDescription {
+    class func insertNewObject<T: NSManagedObject>(
+        into context: NSManagedObjectContext,
+        for request: NSFetchRequest<T>
+    ) -> T {
+        let obj = T(context: context)
```

Comment:
So this will do the same as the old `insertNewObject` function? Good to know

Reply:
exactly!